plugins/dell-dock/fu-dell-dock-i2c-ec.c

```diff
@@ -524,6 +535,12 @@ fu_dell_dock_ec_modify_lock (FuDevice *device,
 		 fu_device_get_name (device),
 		 fu_device_get_id (device));
 
+	if (unlocked)
+		self->dock_unlock_status |= 1 << target;
+	else
+		self->dock_unlock_status &= ~(1 << target);
```

Comment:
I know what you're doing, but is there any BIT_SET and BIT_CLEAR macros we can use?

Reply:
How about locally defining these in the file:
```
#define BIT(x)                          (1 << x)
#define BIT_SET(x,y)		(x |= (y))
#define BIT_CLEAR(x,y)		(x &= (~y))
```

And then it turns into
```
if (unlocked)
        BIT_SET (self->dock_unlock_status, BIT(target));
else
        BIT_CLEAR (self->dock_ulock_status, BIT(target));
```

Reply:
I've added that now.

Reply:
Isn't

    #define BIT_SET(x,y)			(x |= (1<<(y)))
    #define BIT_CLEAR(x,y)			(x &= (~1<<(y)))

the usual form?

Reply:
That way is fine to me too, just adjusted.